snakes_cafe.py

```diff
@@ -0,0 +1,65 @@
+
+menu = """**************************************
+**    Welcome to the Snakes Cafe!   **
+**    Please see our menu below.    **
+**
+** To quit at any time, type "quit" **
+**************************************
+
+Appetizers
+----------
+Wings
+Cookies
+Spring Rolls
+
+Entrees
+-------
+Salmon
+Steak
+Meat Tornado
+A Literal Garden
+
+Desserts
+--------
+Ice Cream
+Cake
+Pie
+
+Drinks
+------
+Coffee
+Tea
+Blood of the Innocent
```

Comment:
This works great. However if you add an item to the menu, this requires that you change the menu in multiple places. Try generating this menu print-out instead of hard coding it.